ccd-definition/FixedLists.json

```diff
@@ -496,6 +496,20 @@
     "ListElement": "Don't know",
     "DisplayOrder": 3
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ProceedingType",
+    "ListElementCode": "NOTICE_OF_PROCEEDINGS_FOR_PARTIES",
+    "ListElement": "Notice of proceedings for parties (c6)",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ProceedingType",
+    "ListElementCode": "NOTICE_OF_PROCEEDINGS_FOR_NON_PARTIES",
+    "ListElement": "Notice of proceedings for parties (c6a)",
```

Comment:
does this need to be notice of proceedings for non parties?

Reply:
Good spot thanks